cmd/context/create.go

```diff
@@ -172,8 +172,19 @@ func (c *ContextCommand) UseContext(ctx context.Context, ctxOptions *ContextOpti
 			return err
 		}
 
-		if !hasAccess {
-			return oktetoErrors.UserError{E: fmt.Errorf("namespace '%s' not found on context '%s'", ctxOptions.Namespace, ctxOptions.Context),
+		// if using a new context, our cached namespace may have been removed
+		// so swap over to the personal namespace instead of erroring
+		if !hasAccess && !ctxOptions.FromNamespace {
```

Comment:
can we simplify the boolean logic here? maybe something like:
```
if !hasAccess {
  if ctxOptions.FromNamespace {
    // Return error here
  }
  // Handle default here
}
```

Reply:
Done